library/alloc/src/vec.rs

```diff
@@ -1610,8 +1610,6 @@ impl<T: Clone> Vec<T> {
     /// vec.extend_from_slice(&[2, 3, 4]);
     /// assert_eq!(vec, [1, 2, 3, 4]);
     /// ```
```

Comment:
```suggestion
    /// ```
    ///
    /// [`extend`]: Vec::extend
```

Reply:
Wait, that's the only thing this patch does. I thought you say it can be removed already?

Reply:
No, there is an issue open to _add_ this feature (https://github.com/rust-lang/rust/issues/59039), but it is not yet implemented. My suggested change goes from `#method.extend` to the associated function.